File: accepted-solutions/oddities/oddities.cpp
#include <iostream>

using namespace std;

int main()
{
    int num, x;

    cin >> num;
    
    do {
        cin >> x;
        
        if ( x & 1 )
            cout << x << " is odd" << endl;
        
        else
            cout << x << " is even" << endl;
        
        } while( --num > 0 );
}

File: accepted-solutions/oddities/oddities.java
import java.util.Scanner;

public class oddities {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int num = scan.nextInt();

        do {
            int x = scan.nextInt();

            if ( ( x & 1 ) != 0 ) {
                System.out.println(x + " is odd");
            }
            else {
                System.out.println(x + " is even");
            }
        } while (--num > 0);
    }
}
